doc/conf.py

```diff
@@ -16,37 +16,37 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('../openslide'))
-sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath("../openslide"))
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
Let's not do this one, please.  The codebase is pretty consistent about using single quotes.  (And the change makes this diff pretty hard to read.)

Reply:
skip-string-normalization worked, so we can keep the single quotes, for now...